share/spice/cricket/cricket.js

```diff
@@ -0,0 +1,159 @@
+(function (env) {
+    "use strict";
+    var SCORECARD_ENDPOINT = 'https://query.yahooapis.com/v1/public/yql?q=select%20*%20from%20cricket.scorecard%20where%20match_id%3D{$1}&format=json&env=store%3A%2F%2F0TxIGQMQbObzvU4Apia0V0&callback=',
```

Comment:
@moollaza Yahoo rate limits the YQL APIs per IP per day: http://stackoverflow.com/questions/9346582/what-is-the-query-limit-on-yahoos-finance-api

Anyway I will make it internal API call and later decide on upgrading as it scales.

Also we can't cache Live Score API call